File: src/BukkitPE/level/particle/TerrainParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\block\Block;
use BukkitPE\math\Vector3;

class TerrainParticle extends GenericParticle{
	public function __construct(Vector3 $pos, Block $b){
		parent::__construct($pos, Particle::TYPE_TERRAIN, ($b->getDamage() << 8) | $b->getId());
	}
}

File: src/main/java/net/BukkitPE/level/particle/TerrainParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.block.Block;
import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class TerrainParticle extends GenericParticle {
    public TerrainParticle(Vector3 pos, Block block) {
        super(pos, Particle.TYPE_TERRAIN, (block.getDamage() << 8) | block.getId());
    }
}
